src/component/App.js

```diff
@@ -0,0 +1,57 @@
+import Component from './Component.js';
+import Header from './Header.js';
+import Filter from './Filter.js';
+import TodoList from './TodoList.js';
+import todos from '../../data/todo.js';
+import AddTodo from './AddTodo.js';
+
+class App extends Component {
+    render() {
+        const dom = this.renderDOM();
+
+        const header = new Header();
+        const headerDOM = header.render();
+        const main = dom.querySelector('main');
+        dom.insertBefore(headerDOM, main);
+
+        const addToDo = new AddTodo({
+            onAdd: (newToDo) => {
+                todos.push(newToDo);
+                todoList.update({ todos });
+            }
+        });
+        const addToDoDOM = addToDo.render();
+        main.appendChild(addToDoDOM);
+
+        const filter = New Filter({
```

Comment:
This throws ```unexpected identifier``` on page load because you have ```const filter = New Filter()``` and the way to instantiate a new class is ```const filter = new Filter()```